dev/run-tests.py

```diff
@@ -231,7 +231,7 @@ def exec_maven(mvn_args=()):
     """Will call Maven in the current directory with the list of mvn_args passed
     in and returns the subprocess for any further processing"""
 
-    run_cmd([os.path.join(SPARK_HOME, "build", "mvn")] + mvn_args)
+    run_cmd([os.path.join(SPARK_HOME, "build", "mvn")] + ["--force"] + mvn_args)
```

Comment:
Technically, the `--force` can be in the first list, but it doesn't really matter.